metalog.js

```diff
@@ -171,6 +176,10 @@ class Logger extends events.EventEmitter {
 
   close() {
     if (!this.active) return;
+    if (!this.fsEnabled) {
+      this.emit('close');
```

Comment:
Maybe `close` event should be emitted in `process.nextTick` as on [line 149](https://github.com/metarhia/metalog/pull/81/commits/679ec795682c3215d5d57b1290c9a2a12759dc93#diff-de15682abd39643d598f22220edccd6cR149) of this file?

Reply:
As stated in the related issue https://github.com/metarhia/metalog/issues/80:

> as well as `close` event should be emitted immediately/synchronously on `Logger.close()`.

I've used `process.nextTick()` when emitting `'open'` event only because it's emitted inside the `Logger#open()` method that is being called in the constructor of that class, meaning that it's not possible to attach an `'open'` event listener before the event is emitted, if it is emitted synchronously.